src/status_im/subs/communities_test.cljs

```diff
@@ -459,3 +459,49 @@
                                       :loading?    checking-permissions?
                                       :img-src     token-image-eth}]]}
            (rf/sub [sub-name community-id])))))
+
+(h/deftest-sub :communities/airdrop-account
+  [sub-name]
+  (testing "returns airdrop account"
+    (swap! rf-db/app-db assoc
+      :communities/airdrop-address
+      "0x1"
+      :wallet {:accounts {"0x1" {:address "0x1"
+                                 :color   :blue
+                                 :name    "account1"}
+                          "0x2" {:address "0x2"
+                                 :color   :orange
+                                 :name    "account2"}}})
+    (is (= {:address                   "0x1"
```

Comment:
I tested, and `match?` works here, it's better than `is` for larger data structures.